static/rfcs/10/overview.msc

```diff
@@ -0,0 +1,41 @@
+# Sequence diagram for Waku v2 (WakuMessage, WakuData, Relay, Store, Filter)
+msc {
+  hscale="1",
+  wordwraparcs=true;
+
+  a [label="A [relay]"],
+  b [label="B [relay(topic1)]"],
+  c [label="C [relay(topic2)]"],
+  d [label="D [relay(topic1), store(topic1)]"],
```

Comment:
Might be good to show that `D` needs to have `filter` mounted as well in order to respond to `F`. Can then illustrate different filter roles ("light" or "full") in the filter spec.

Reply:
Addressed